applications/ULFapplication/custom_elements/ulf_axisym.h

```diff
@@ -221,47 +221,45 @@ class UlfFrac3DSwimming
 
 private:
     ///@name Static Member Variables
-    ///@{
-    //static boost::numeric::ublas::bounded_matrix<double,3,3> msMassFactors;
-    //static boost::numeric::ublas::bounded_matrix<double,3,2> msDN_Dx;
-    //static array_1d<double,3> msN; //dimension = number of nodes
-    //static Matrix msDN_DX;
-    //static Matrix msMassFactors;
-    //static array_1d<double,2> ms_vel_gauss; //dimesion coincides with space dimension
-    //static array_1d<double,3> ms_temp_vec_np; //dimension = number of nodes
-    //static array_1d<double,3> ms_u_DN;
-
-
-    //static boost::numeric::ublas::bounded_matrix<double,3,6> msB;
-    //static boost::numeric::ublas::bounded_matrix<double,3,3> ms_constitutive_matrix;
-    //static boost::numeric::ublas::bounded_matrix<double,3,6> ms_temp;
-    //static array_1d<double,6> ms_temp_vec;
-
 
+    ///@{
     ///@}
     ///@name Member Variables
     ///@{
     double mA0; //stores area at the beginning of time-step
-
-
+    double r0;// stores the radius at the beginning of time-step
 
     ///@}
-    ///@name Private Operators
+    ///@name Serialization
     ///@{
-    void CalculateLumpedMass();
-    void PressureStep(MatrixType& rLeftHandSideMatrix, VectorType& rRightHandSideVector, ProcessInfo& rCurrentProcessInfo);
-    void VelocityStep(MatrixType& rLeftHandSideMatrix, VectorType& rRightHandSideVector, ProcessInfo& rCurrentProcessInfo);
-    //inline void CalculateGeometryData(Matrix& msDN_DX, Vector& N, double& Area)
-    //inline void CalculateGeometryData(boost::numeric::ublas::bounded_matrix<double,4,3>& DN_DX, array_1d<double,4>& N, double& Area);
+    friend class Serializer;
+
+    // A private default constructor necessary for serialization
+    UlfAxisym() : Element()
+    {
+    }
+
+    virtual void save(Serializer& rSerializer) const
+    {
+        KRATOS_SERIALIZE_SAVE_BASE_CLASS(rSerializer, Element);
```

Comment:
It is important to save and load all your member variables in serializer

Reply:
I didnt understand this one. What am I missing in the serializer? Which variable?